module/component/component.go

```diff
@@ -157,15 +157,23 @@ func NewComponentManagerBuilder() ComponentManagerBuilder {
 	return &componentManagerBuilderImpl{}
 }
 
-// AddWorker adds a ComponentWorker closure to the ComponentManagerBuilder
 // All worker functions will be run in parallel when the ComponentManager is started.
 // Note: AddWorker is not concurrency-safe, and should only be called on an individual builder
-// within a single goroutine.
+// within a single goroutine.// AddWorker adds a ComponentWorker closure to the ComponentManagerBuilder
+
 func (c *componentManagerBuilderImpl) AddWorker(worker ComponentWorker) ComponentManagerBuilder {
 	c.workers = append(c.workers, worker)
 	return c
 }
 
+// AddWorkers adds n number of workers for the ComponentManager.
+func (c *componentManagerBuilderImpl) AddWorkers(n int, worker ComponentWorker) ComponentManagerBuilder {
```

Comment:
Looks like you removed this from the interface. Did you intend to revert these changes too?

Reply:
https://github.com/onflow/flow-go/pull/4553/commits/c1d3b2e0922e62309f67db7615671e24155de1ed